contracts/src/systems/craft.cairo

```diff
@@ -0,0 +1,71 @@
+#[system]
+mod Craft {
```

Comment:
i wouldnt call this crafting, crafting is an entirely different part of the game where we can create different materials from resources.